src/promptflow/promptflow/_sdk/_service/entry.py

```diff
@@ -119,16 +119,44 @@ def validate_port(port, force_start):
         print(f"Start Prompt Flow Service on http://localhost:{port}, version: {get_promptflow_sdk_version()}")
         waitress.serve(app, host="127.0.0.1", port=port)
     else:
-        # Set host to localhost, only allow request from localhost.
-        cmd = ["waitress-serve", f"--listen=127.0.0.1:{port}", "promptflow._sdk._service.entry:get_app"]
         # Start a pfs process using detach mode. It will start a new process and create a new app. So we use environment
         # variable to pass the debug mode, since it will inherit parent process environment variable.
         if platform.system() == "Windows":
-            # For windows, use creationflags=subprocess.CREATE_NEW_PROCESS_GROUP to create a new process group. But
-            # if you close the terminal which start the cmd, the child process will stop since it will send a close
-            # signal to process group started in the terminal.
-            subprocess.Popen(cmd, stdout=subprocess.DEVNULL, creationflags=subprocess.CREATE_NEW_PROCESS_GROUP)
+            try:
+                import win32api
+                import win32con
+                import win32process
+            except ImportError as ex:
+                raise UserErrorException(
+                    f"Please try to run 'pip install pywin32' to start prompt flow service with pywin32. {ex}"
```

Comment:
Let's use this wording: 

Please install pywin32 by 'pip install pywin32' and retry. prompt flow service start depends on pywin32.